subprojects/smoke-test/build.gradle.kts

```diff
@@ -121,7 +128,11 @@ tasks {
 
     register<SmokeTest>("gradleBuildSmokeTest") {
         description = "Runs Smoke tests against the Gradle build"
-        configureForSmokeTest(gradleBuildCurrent)
+        configureForSmokeTest(gradleBuildCurrent.map {
+            project.fileTree(it.outputDirectory) {
+                exclude("subprojects/*/src/**")
```

Comment:
I think we should also exclude some more files: https://github.com/gradle/gradle/blob/54460ce7b90882079b699e05537f9c09e0126ffe/.teamcity/src/main/kotlin/configurations/VcsTriggers.kt#L21-L24